README.md

```diff
@@ -109,3 +122,5 @@ See all meetups and events [here](https://hacktoberfest.digitalocean.com/events)
 [![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](http://creativecommons.org/publicdomain/zero/1.0)
 
 To the extent possible under law, [Otacilio Maia](https://github.com/OtacilioN) has waived all copyright and related or neighbouring rights to this work.
+
+
```

Comment:
Can you remove this duplicated empty line at the end?

Reply:
Yes. now I removed it.

Reply:
Yes. now I removed it.